packages/connect/src/types/api/cardano/index.ts

```diff
@@ -140,13 +140,21 @@ export interface CardanoPoolParameters {
     metadata: CardanoPoolMetadata;
 }
 
+export interface CardanoDRep {
+    type: PROTO.CardanoDRepType;
+    keyHash?: string;
+    scriptHash?: string;
+}
+
 export interface CardanoCertificate {
     type: PROTO.CardanoCertificateType;
     path?: DerivationPath;
     pool?: string;
     poolParameters?: CardanoPoolParameters;
     scriptHash?: string;
     keyHash?: string;
+    deposit?: string;
+    drep?: CardanoDRep;
```

Comment:
```suggestion
    dRep?: CardanoDRep;
```
(it would be consistent with ledgerjs code etc.)